Engineering/Templates/PullRequestTemplate.md

```diff
@@ -0,0 +1,44 @@
+Thank you for contributing.
+Before submitting this PR, please make sure:
+
+-------------------------------------
+* [ ] I have followed the guidelines in [Contributing as Developers](https://github.com/microsoft/code-with-engineering-playbook/blob/master/CONTRIBUTING.md)
+* [ ] I have added an explanation of what my changes do and why I would like you to include them
+* [ ] I have updated the documentation accordingly.
```

Comment:
Lines 7-11. Inconsistent line endings (periods). I'd recommend including periods on all items in the checklist.

If for some reason we decide no periods on checklist items, we should remove for consistency.